preview-src/icon.tsx

```diff
@@ -12,6 +12,7 @@ export const checkIcon = <Icon src={require('../resources/icons/check.svg')} />;
 export const plusIcon = <Icon src={require('../resources/icons/plus.svg')} />;
 export const deleteIcon = <Icon src={require('../resources/icons/delete.svg')} />;
 export const pendingIcon = <Icon src={require('../resources/icons/dot.svg')} />;
+export const loadingIcon = <Icon src={require('../resources/icons/small_dot.svg')} />;
```

Comment:
I think we can tweak the existing `pendingIcon`, it looks like it just has the wrong `viewBox`

Reply:
Good point, the smaller icon looks better in the status checks area too.

Reply:
Fixed